src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/system_request_test.cc

```diff
@@ -229,6 +229,125 @@ TEST_F(SystemRequestTest, Run_RequestTypeDisallowed_SendDisallowedResponse) {
   command->Run();
 }
 
+TEST_F(SystemRequestTest, Run_FileWithBinaryDataSaveUnsuccessful_GenericError) {
+  std::vector<uint8_t> binary_data;
+  binary_data.push_back(0x1);
+  binary_data.push_back(0x2);
+  binary_data.push_back(0x4);
+
+  const std::string file_name = "file_name";
+  MessageSharedPtr msg = CreateIVSUMessage();
+  const mobile_apis::RequestType::eType request_type =
+      mobile_apis::RequestType::CLIMATE;
+  (*msg)[am::strings::msg_params][am::strings::request_type] = request_type;
+  (*msg)[am::strings::msg_params][am::strings::file_name] = file_name;
+  (*msg)[am::strings::params][am::strings::binary_data] = binary_data;
+
+  PreConditions();
+
+  EXPECT_CALL(mock_policy_handler_,
+              IsRequestTypeAllowed(kAppPolicyId, request_type))
+      .WillOnce(Return(true));
+
+  const std::string kBinaryFolder = "test_binary_folder";
```

Comment:
@ZhdanovP 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar please take a look 8ed4a05b95d7bfc885e2bad85a6970636ef2e7f8